app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None | str:
+    if command.startswith("mv "):
+        pass
+    else:
+        return "write correct command"
+    source_name, dest_path = command[3:].split(" ")
```

Comment:
You should first check if, after splitting, you have obtained 3 values, and only then can you unpack them like this.

Reply:
You should check  `if command == 'mv' ` condition is much clearer and more concise. It explicitly checks if the command variable is exactly equal to "mv,"